detection-rules/text_bec_unsolicited.yml

```diff
@@ -0,0 +1,32 @@
+name: "BEC from unsolicited sender"
+description: |
+    NLU engine classifies the email body as Business Email Compromise with
+    an urgent tone detected.
+
+    Email is from First-time Sender
```

Comment:
can probably drop this (we don't mention it explicitly in the other first-time sender rules)

```suggestion
```